### DISCUSSION TITLE
Help with SelectMenuEvent

### DISCUSSION THREAD

**Discussion by dellix889** [discord-jda/JDA#2137] [Questions and Help]
Hello, I want to broadcast languages ​​by selecting the language in SelectMenu, but I have such a problem that when User selects a different language, he takes the first one that was chosen from him.

For example:
User the first time select lang "ENG", and user got role lang ENG, and when user want to add eng "FR", user got FR and this ENG,
the problem is when user have ENG, and select FR, this ENG remove role from player... Any help?

**Comment by Splayfery** [comment]
You can add .setRequiredRange() to your SelectMenuBuilder that the user can select between more than 1 option. In the SelectMenuInteractionEvent you simply loop your clicked options by creating a for loop with: `for (String value : event.getValues())`
